controllers/clustergroupupgrade_controller.go

```diff
@@ -253,10 +256,15 @@ func (r *ClusterGroupUpgradeReconciler) Reconcile(ctx context.Context, req ctrl.
 				}
 			} else {
 				// Add the needed cluster names to upgrade to the appropriate placement rule.
-				err = r.addClustersToPlacementRule(ctx, clusterGroupUpgrade)
+				reconcileSooner, err := r.remediateCurrentBatch(ctx, clusterGroupUpgrade)
 				if err != nil {
 					return ctrl.Result{}, err
 				}
+				if reconcileSooner == true {
+					requeueAfter := 30 * time.Second
```

Comment:
Better to have the 30 as a constant for "fast requeue", but this can be done in a subsequent cleanup.

Reply:
Yes, there are a couple of requeue values that should be made constants.